From 8657727b58064d88727467ee29333a02c393ab2a Mon Sep 17 00:00:00 2001
From: Chenhui Zhao <chenhui.zhao@nxp.com>
Date: Wed, 15 Oct 2025 19:10:15 +0800
Subject: [PATCH] correct path to tar binary

correct the tar path in TAR_BINARY which should be the path in the embedded
rootfs instead of the path in the host filesystem.

Upstream-Status: Inappropriate [OE/crosscompile specific]

Signed-off-by: Chenhui Zhao <chenhui.zhao@nxp.com>
---
 CMakeLists.txt | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9b36de44..d9b18f31 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -582,10 +582,7 @@ target_link_libraries(sysrepo ${CMAKE_THREAD_LIBS_INIT})
 set(CMAKE_REQUIRED_LIBRARIES pthread)
 
 # tar
-find_program(TAR_BINARY "tar")
-if(NOT TAR_BINARY)
-    message(FATAL_ERROR "tar binary was not found.")
-endif()
+set(TAR_BINARY "/usr/bin/tar")
 
 # generate files
 configure_file("${PROJECT_SOURCE_DIR}/src/config.h.in" "${PROJECT_BINARY_DIR}/config.h" ESCAPE_QUOTES @ONLY)
-- 
2.34.1

